pkgs/development/python-modules/nodriver/default.nix

```
@@ -0,0 +1,48 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, deprecated
+, mss
+, websockets
+, setuptools
+}:
+
+buildPythonPackage {
+  pname = "nodriver";
+  version = "0.32";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "ultrafunkamsterdam";
+    repo = "nodriver";
+    rev = "cbb8e9ac26b70310080464d2ad43a1e2de2bfbaa";
+    hash = "sha256-FPZ+SQ7ToJMC8Wg3cAhrnuA+rqtpnjj/yLaFHIYWg8w=";
+  };
+
+  disabled = pythonOlder "3.9";
+
+  dependencies = [
+    deprecated
+    mss
+    websockets
+  ];
+
+  build-system = [
+    setuptools
+  ];
+
+  pythonImportsCheck = [
+    "nodriver"
+  ];
+
```

```suggestion
  # no tests
  doCheck = false;
```
It is recommended to explicitly disable tests in python modules.

> Make sure the tests are enabled using for example [pytestCheckHook](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pytestcheckhook) and, in the case of libraries, are passing for all interpreters. If certain tests fail they can be disabled individually. Try to avoid disabling the tests altogether. In any case, when you disable tests, leave a comment explaining why.

https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#contributing-guidelines-contributing-guidelines